ai-ml/jupyterhub/addons.tf

```diff
@@ -0,0 +1,126 @@
+module "eks_blueprints_kubernetes_addons" {
+  source = "github.com/aws-ia/terraform-aws-eks-blueprints//modules/kubernetes-addons?ref=v4.25.0"
```

Comment:
replace this module with https://github.com/awslabs/data-on-eks/blob/47076477f1d691c0aaa7ca4a04ac215bd7e91837/analytics/terraform/spark-k8s-operator/addons.tf#L53

Now you can use the latest tag from the addons repo instead of hash. Check the addons module for latest tag